src/services/alert/index.js

```diff
@@ -0,0 +1,59 @@
+const hooks = require('feathers-hooks-common');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'feathers-hooks-common'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4499680)